samples/bluetooth/unicast_audio_client/src/main.c

```diff
@@ -724,8 +724,16 @@ static void unicast_client_location_cb(struct bt_conn *conn,
 	printk("dir %u loc %X\n", dir, loc);
 }
 
+static void available_contexts_cb(struct bt_conn *conn,
+				  enum bt_audio_context snk_ctx,
+				  enum bt_audio_context src_ctx)
+{
+	printk("snk ctx %u src ctx %u\n", snk_ctx, src_ctx);
+}
+
 const struct bt_audio_unicast_client_cb unicast_client_cbs = {
-	.location = unicast_client_location_cb
+	.location = unicast_client_location_cb,
+	.available_contexts = available_contexts_cb
```

Comment:
nitpick, put comma to not touch this line when `unicast_client_cbs` will be extended with another callback ^^